src/pages/iou/steps/MoneyRequestConfirmPage.js

```diff
@@ -74,6 +75,14 @@ function MoneyRequestConfirmPage(props) {
         [props.iou.participants, props.personalDetails],
     );
 
+    useEffect(() => {
+        // We want to load categories if this is an expense report on a policy expense chat
+        const policyExpenseChat = _.find(participants, (participant) => participant.isPolicyExpenseChat);
```

Comment:
I would use `isPolicyExpenseChat` instead here.

Reply:
@yuwenmemon Are you referring to `ReportUtils.isPolicyExpenseChat()`? I don't think we can use that here because the `participant` isn't actually a report object. It has [this](https://github.com/Expensify/App/blob/main/src/pages/iou/steps/MoneyRequestConfirmPage.js#L37-L43) shape.

Reply:
No, for the variable name I mean

Reply:
Ah hmm, it isn't really a boolean either. It returns the participant object with that shape, and we access the policyID from it later. So it might be kinda strange to access it like `isPolicyExpenseChat.policyID`.